src/Symfony/Component/Console/Attribute/Argument.php

```diff
@@ -41,10 +43,12 @@ class Argument
      * @param array<string|Suggestion>|callable(CompletionInput):list<string|Suggestion> $suggestedValues The values used for input completion
      */
     public function __construct(
-        public string $description = '',
+        string|callable $description = '',
```

Comment:
I hope nobody has a description like `"define"` as this would be detected as a callable and throw an error when called with no argument.

The type could be updated to `string|\Closure`, but that would only work with PHP 8.5.

Reply:
Hah, I've just tested it, you're right…

```php
#[Argument(description: 'define')]
```

```
ArgumentCountError {#55
  #message: "define() expects at least 2 arguments, 0 given"
  #code: 0
  #file: "./vendor/symfony/console/Attribute/Argument.php"
  #line: 45
```

Does it make this PR breaking backwards compatibility?